slides/src/content/a11y.md

```diff
@@ -215,6 +217,36 @@ For example in your header div:
 
 ---
 <!-- .slide: id="a11y" -->
+## Accessible Development - Defining Page Sections
+###CSS Clip Attribute
```

Comment:
We aren't using sub-headings in other slides - rename this slide (and the ones that follow) to just be "CSS Clip Attribute" maybe?